fe-nc-news/src/Components/ArticleDetails.jsx

```diff
@@ -2,19 +2,28 @@ import { useState, useEffect } from "react";
 import { Link, useParams } from "react-router-dom";
 import { apiClient } from "../utils/APIclient";
 import { ArticleDetailsCard } from "./ArticleDetailsCard";
+import { Comments } from "./Comments";
+Comments;
 export const ArticleDetails = () => {
   const { article_id } = useParams();
   const [articleDetails, setArticleDetails] = useState({});
+  const [comments, setComments] = useState([]);
 
   useEffect(() => {
     apiClient.get(`/articles/${article_id}`).then((response) => {
       setArticleDetails(response.data.article);
     });
   }, []);
 
+  useEffect(() => {
+    apiClient.get(`/articles/${article_id}/comments`).then((response) => {
+      setComments(response.data.comments);
+    });
+  }, []);
+
```

Comment:
There's a lot going on here. This SingleArticle component is now in charge of fetching article details and also comments. I would move any comments logic into the existing Comments component that you have